client/src/app/site/core/core.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { MotionsRoutingModule } from './core-routing.module';
+import { SharedModule } from '../../shared/shared.module';
+import { PrivacyPolicyComponent } from './components/privacy-policy/privacy-policy.component';
+import { StartComponent } from './components/start/start.component';
+import { LegalNoticeComponent } from './components/legal-notice/legal-notice.component';
+
+@NgModule({
+    imports: [CommonModule, MotionsRoutingModule, SharedModule],
+    declarations: [PrivacyPolicyComponent, StartComponent, LegalNoticeComponent]
+})
+export class CoreModule {}
```

Comment:
we have two `CoreModule`s now. As discussed earlier, avoiding repeating module names would be highly appreciated.
Easy alternatives would be "main" or "master".

we could also rename the 'site' folder to something else or include multiple feature-folders.

i.e.

- core
- shared
- access
  - start
  - login
  - privacy-policy
  - legal-notie
- projector
  - ( ... )
- main
  - motions
  - agenda
  - ( ... )

Reply:
Isn't "master" a bit discriminating? (see https://bugs.python.org/issue34605 for further reference).

BTT:  I'm all for renaming the `core` in sites to something else, like `corn-hub`, `nitty-gritty` or `heart` or `common`, please do not just add a prefix, like `os-core`. The reason for keeping `core` in the main folder for me would be, that it may hurt us in a later upgrade process, since the standard-folders are then renamend.